backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/SendMail.kt

```diff
@@ -1,5 +1,19 @@
 package app.ehrenamtskarte.backend.mail
 
-fun sendMail(to: String, subject: String, message: String) {
-    println("SENDING PSEUDO MAIL TO $to\nSUBJECT: $subject\n$message")
+import app.ehrenamtskarte.backend.config.SmtpConfig
+import org.simplejavamail.email.EmailBuilder
+import org.simplejavamail.mailer.MailerBuilder
+
+fun sendMail(smtpConfig: SmtpConfig, fromName: String, to: String, subject: String, message: String) {
+    MailerBuilder.withSMTPServer(smtpConfig.host, smtpConfig.port, smtpConfig.username, smtpConfig.password)
```

Comment:
~~I think we need here: `withTransportStrategy "SMTP_TLS"`~~

~~Else it uses port 25 which is unencrypted~~

Reply:
Oke probably not if we specify port 587

Reply:
It worked, so I guess, the config is correct :D